openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -74,18 +74,19 @@ export default function init(){
     }
 
     renderInstantSearchResults = function(q) {
-        var facet_value = searchFacets[localStorage.getItem('facet')];
-        var url, facet;
+        const facet_value = searchFacets[localStorage.getItem('facet')];
+        // Not implemented; also, this call is _expensive_ and should not be done!
+        if (facet_value === 'inside') return;
```

Comment:
I'd prefer us to always wrap the if statements in curly braces but clearly our eslinter is not doing that right now and that's a conversation for a later date.